sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/Watch.java

```diff
@@ -1074,6 +1106,40 @@ public HashCode decode(InputStream is) throws IOException {
     }
   }
 
+  private static class SnappyCoder<T> extends StructuredCoder<T> {
```

Comment:
<!--new_thread; commit:b04f5ed31c0be1f231dd7c1613ea50d31616b127; resolved:0-->
Should we take make this public (and it's own Java file) ? This might be useful for other transforms.

Reply:
Done.